modules/power-monitoring-configuring-kepler-redfish.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+
+// * power_monitoring/configuring-power-monitoring.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="power-monitoring-configuring-kepler-redfish_{context}"]
+= Configuring {PM-kepler} to use Redfish
+
+You can configure {PM-kepler} to use Redfish as the source for running or hosting containers. {PM-kepler} can then monitor the power usage of these containers.
+
+.Prerequisites
+* You have access to the {product-title} web console.
+* You are logged in as a user with the `cluster-admin` role.
+* You have installed the {PM-operator}.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, go to *Operators* -> *Installed Operators*.
+
+. Click *{PM-title-c}* from the *Installed Operators* list and go to the *{PM-kepler}* tab.
+
+. Click *Create {PM-kepler}*. If you already have a {PM-kepler} instance created, click *Edit Kepler*.
+
+. Configure `.spec.exporter.redfish` of the {PM-kepler} instance by specifying the mandatory `secretRef` field. You can also configure the optional `probeInterval` and `skipSSLVerify` fields to meet your needs.
++
+.Example output
+[source,yaml]
+----
+apiVersion: kepler.system.sustainable.computing.io/v1alpha1
+kind: Kepler
+metadata:
+  name: kepler
+spec:
+  exporter:
+    deployment:
+# ... 
+    redfish:
+      secretRef: <name of the secret> required <1>
+      probeInterval: 60s <2>
+      skipSSLVerify: false <3>
+# ...
+----
+<1> Required: Specifies the name of the secret that contains the credentials for accessing the Redfish server.
+<2> Optional: Controls the frequency at which the power information is queried from Redfish. The default value is `60s`.
+<3> Optional: Controls if {PM-kepler} skips verifying the Redfish server certificate. The default value is `false`.
++
+[NOTE]
+====
+After {PM-kepler} is deployed, the `openshift-power-monitoring` namespace is created.
+====
+. Create the `redfish.csv` file with the following data format: `<your_kubelet_node_name>, <redfish_username>, <redfish_password>, https://<redfish_ip_or_hostname>/`.
++
+.Example output
+[source,csv]
+----
+control-plane,dummyuser,dummypass,https://redfish.nodes.example.com
+worker-1,dummyuser,dummypass,https://redfish.nodes.example.com
+worker-2,dummyuser,dummypass,https://another.redfish.nodes.example.com
+----
+. Create the secret under the `openshift-power-monitoring` namespace. You must create the secret with the following conditions:
+* The secret type is `Opaque`.
+* The credentials are stored under the `redfish.csv` key in the `data` field of the secret.
++
```

Comment:
This unordered list in the middle of the command is making the indentation get off for the command block that follows. You can use an [open block](https://docs.asciidoctor.org/asciidoc/latest/blocks/open-blocks) (`--`) to fix:
```suggestion
. Create the secret under the `openshift-power-monitoring` namespace. You must create the secret with the following conditions:
+
--
* The secret type is `Opaque`.
* The credentials are stored under the `redfish.csv` key in the `data` field of the secret.
--
+
```